sdcm/logcollector.py

```diff
@@ -220,7 +220,11 @@ class MonitoringStack(BaseLogEntity):
 
     @staticmethod
     def get_monitoring_base_dir(node):
-        return node.parent_cluster.monitor_install_path_base
+        from sdcm.cluster import BaseMonitorSet
```

Comment:
why the import is here?

Reply:
@aleksbykov if it's cause of cyclic dependency, please put a comment saying so

Reply:
added